### PR TITLE
[ntfs2gtfs] write agency.txt

### PR DESCRIPTION
**Pull request by datanel**

### CODE REVIEW

File: src/gtfs/mod.rs [hunk 1]
```diff
@@ -23,11 +23,46 @@ use collection::CollectionWithId;
...
+impl From<objects::Network> for Agency {
+    fn from(n: objects::Network) -> Agency {
+        Agency {
+            id: Some(n.id.clone()),
+            name: n.name.clone(),
+            url: n
+                .url
+                .clone()
+                .unwrap_or("http://www.navitia.io/".to_string()),
+            timezone: n.timezone.clone().or(Some("Europe/Paris".to_string())),
```
Comment (TeXitoi) [inline src/gtfs/mod.rs]: Are you sure of these defaults? @prhod ?

File: src/gtfs/mod.rs [hunk 2]
```diff
@@ -23,11 +23,46 @@ use collection::CollectionWithId;
...
+    phone: Option<String>,
+    #[serde(rename = "agency_email")]
+    email: Option<String>,
+}
+
+impl From<objects::Network> for Agency {
+    fn from(n: objects::Network) -> Agency {
+        Agency {
+            id: Some(n.id.clone()),
+            name: n.name.clone(),
```
Comment (TeXitoi) [inline src/gtfs/mod.rs]: All these clones should be useless as you have the ownership of a Network.

File: src/gtfs/write.rs [hunk 3]
```diff
@@ -13,3 +13,95 @@
...
+            id: Some(n.id.clone()),
+            name: n.name.clone(),
+            url: n
+                .url
+                .clone()
+                .unwrap_or("http://www.navitia.io/".to_string()),
+            timezone: n.timezone.clone().or(Some("Europe/Paris".to_string())),
+            lang: n.lang.clone(),
+            phone: n.phone.clone(),
+            email: None,
```
Comment (TeXitoi) [inline src/gtfs/write.rs]: You should use the from implementation here.

File: src/gtfs/mod.rs (hunk 2 above)
Comment (datanel) [reply src/gtfs/mod.rs]: I have 
```
error[E0507]: cannot move out of borrowed content
  --> src/gtfs/write.rs:31:22
   |
31 |             id: Some(n.id),
   |                      ^ cannot move out of borrowed content
```

File: src/gtfs/mod.rs (hunk 1 above)
Comment (prhod) [reply src/gtfs/mod.rs]: These are the actual defaults for Navitia.io, but they shouldn't be used in most cases.
Do you have a better idea ?

File: src/gtfs/mod.rs [hunk 4]
```diff
@@ -23,11 +23,46 @@ use collection::CollectionWithId;
...
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+struct Agency {
+    #[serde(rename = "agency_id")]
+    id: Option<String>,
+    #[serde(rename = "agency_name")]
+    name: String,
+    #[serde(rename = "agency_url")]
+    url: String,
+    #[serde(rename = "agency_timezone")]
+    timezone: Option<String>,
```
Comment (TeXitoi) [inline src/gtfs/mod.rs]: Must be a String, not an option according to the spec.

**Pull request merged**